BFD-2736: Cleanup primary BFD README (#1822)
CMSgov/beneficiary-fhir-data: README.md

> Beneficiary FHIR Data Server (BFD)

Reviewer: I don't feel _too_ strongly about this, but... this GitHub Project and Repository covers more than _just_ the BFD Server. What do you think?
```suggestion
Beneficiary FHIR Data System (BFD)
```

Author: Yeah that makes sense. The original intent of the while project is to support the API server, but the repo is certainly the conglomeration of all the parts to get that running, so it makes sense to call it a system.

====================================

## About

This project contains modules and documentation in support of the Beneficiary FHIR Data (BFD) Server. 

> The BFD Server is an API designed to serve Medicare beneficiaries' demographic, enrollment, and claims data in [FHIR](https://www.hl7.org/fhir/overview.html) format.

Reviewer: While you're in here... indeed, it is a minor inconvenience, but the good people at Health Level Seven International would be pleased if we followed their [trademark guidance](https://www.hl7.org/fhir/license.html#trademark).

>  [..] This means that for any use of the trademarks ("FHIR" and the FHIR Icon) by parties other than HL7 and it's affiliates:
> [..]
> * When referencing the FHIR® standard in a website, document, presentation, or otherwise, please in a place of prominence refer to it as the "HL7® FHIR® standard". In subsequent uses, please refer to it as the "FHIR® standard" or "FHIR®", using the ® symbol as often as is practical, at least once on each page of printed matter, generally in connection with the first or dominant usage.

I sure hope GitHub doesn't have any issues rendering the US PTO trademark symbol "®" 🤞

```suggestion
The BFD Server is an API designed to serve Medicare beneficiaries' demographic, enrollment, and claims data using the [HL7® FHIR® Standard](https://www.hl7.org/fhir/overview.html) format.
```

## Documentation

### Users

Many useful guides and documentation items can be found on the [BFD Wiki](https://github.com/CMSgov/beneficiary-fhir-data/wiki), hosted in this repo.

This includes information about [making requests to bfd](https://github.com/CMSgov/beneficiary-fhir-data/wiki/Making-Requests-to-BFD), 
[synthetic data](https://github.com/CMSgov/beneficiary-fhir-data/wiki/Synthetic-Data-Guide), and more.

### Contributors and Maintainers

The [BFD Wiki](https://github.com/CMSgov/beneficiary-fhir-data/wiki) contains useful
resources like the [development environment setup guide](https://github.com/CMSgov/beneficiary-fhir-data/wiki/Local-Environment-Setup-for-BFD-Development),
[style guide](https://github.com/CMSgov/beneficiary-fhir-data/wiki/BFD-Code-Style-Guide), and more.

Also for maintainers, the full set of runbooks for various scenarios can be found in [runbooks](https://github.com/CMSgov/beneficiary-fhir-data/tree/master/runbooks).

## Security

We work with sensitive information: do not put any PHI or PII in the public repo for BFD.

If you believe you’ve found or been made aware of a security vulnerability, please refer to 

> the CMS Vulnerability Disclosure Policy (here is a [link](https://www.cms.gov/Research-Statistics-Data-and-Systems/CMS-Information-Technology/CIO-Directives-and-Policies/Downloads/CMS-Vulnerability-Disclosure-Policy.pdf) 

Reviewer: Noticed a spare '(' in the content here, and this is _merely_ a suggestion.
```suggestion
the [CMS Vulnerability Disclosure Policy](https://www.cms.gov/Research-Statistics-Data-and-Systems/CMS-Information-Technology/CIO-Directives-and-Policies/Downloads/CMS-Vulnerability-Disclosure-Policy.pdf)
```

Author: Makes sense, fits the style of the rest of the links. I had just copy/pasted that section without editing it.
